src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -235,18 +228,30 @@ export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
         goodStanding: business.goodStanding
       } as AmalgamatingBusinessIF
 
-      if (!this.amalgamatingBusinesses.find(b => b.identifier === business.identifier)) {
-        this.amalgamatingBusinesses.push(tingBusiness)
+      if (!amalgamatingBusinesses.find((b: any) => b.identifier === business.identifier)) {
```

Comment:
I wonder if we can the set the type here as AmalgamatingBusinessIF Sev?

Reply:
Yes. On it.

Reply:
I mean no. `AmalgamatingBusinessIF` is a union type and identifier is only available when it's used as a lear type.

I could export AmalgamatingLearIF and make this work, but I think it's cleaner to have this single example as any. Open to others' thoughts...

Reply:
Yep, it makes sense. OK, no worries, I'm guessing we can keep it as any since part 2 ticket might use this too.